lib/attach-middleware.js

```diff
@@ -34,4 +34,41 @@ module.exports = function(app, options) {
       });
     },
     logError);
+
+  function shouldExclude(moduleName, excludes) {
+    for (var i = 0; i < excludes.length; i++) {
+      if (minimatch(moduleName, excludes[i])) {
+        return true
+      }
+    }
+    return false
+  }
+  app.post('/filter-entries',bodyParser.urlencoded({
+    limit: '50mb',
+    extended: false,
+    parameterLimit: 10000
+  }),bodyParser.json({
+    limit: '50mb'
+  }),
+    function(req, res) {
+      var entries = req.body['entries[]'] || req.body.entries
+      debugger
+      var seen = {}
+      var appName = options.appName
+      var filteredEntries = entries.filter(function(entry) {
```

Comment:
Can the filter be handled in the middleware so that we only get a list of entries that weren't loaded?